monolith/views/users.py

```diff
@@ -30,8 +31,8 @@ def wall(author_id):
     if author is None:
         abort(404)
 
-    stories = Story.query.filter_by(author_id = author_id)
-    return render_template("wall.html", stories=stories, author=author, 
+    stories = db.session.query(Story).filter(Story.author_id == author_id).filter_by(published=1)
```

Comment:
try with stories =`Story.query.filter_by(author_id = author_id).filter_by(published=1)`

Reply:
Guys it's a comparison

So it is:

stories = db.session.query(Story).filter(Story.author_id == author_id).filter_by(Story.published == 1)